src/ontology/cl-edit.owl

```diff
@@ -30663,6 +30664,20 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A small, peg-like epithelial cell of the fallopian tube (oviduct) epithelium, with very little cytoplasm. These cells are much rarer than the ciliated and secretory epithelial cells of the fallopian tube epithelium and are often found intercalated between them, with high concentrations at the fimbriated, distal end of the tube. They may have a regenerative/stem-cell function. In humans, markers include EPCAM, CD44, and ITGA6."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033014 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033014 "2023-02-02T15:07:12Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "FTE stem-like cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "FTESC")
```

Comment:
Consider adding has_synonym_type "abbreviation". Can refer to CL:4030035 'dental pulp stem cell' as an example.